Descent3/ConfigItem.cpp

```diff
@@ -164,12 +164,14 @@ bool ConfigItem::Create(NewUIGameWindow *parentwnd, int type, int flags, int x,
     Int3();
   }
 
-  if (type == CIT_SLIDER) // slider's need to have the label centered vertically
-    m_tLabel.Create(parentwnd, &UITextItem(label, UICOL_TEXT_NORMAL), x, y + 7, UIF_FIT);
-  else {
+  if (type == CIT_SLIDER) { // slider's need to have the label centered vertically
+    auto item = UITextItem(label, UICOL_TEXT_NORMAL);
```

Comment:
Someone was recommanding `UITextItem item{label, UICOL_TEXT_NORMAL}` which is even shorter and avoids using `auto` https://github.com/kevinbentley/Descent3/issues/47#issuecomment-2064817034